### PR TITLE
VB-2693 Added swagger api and improved code config code.

### PR DESCRIPTION
**Pull request by AledWEvans**
Added swagger api and improved code config code.

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/config/OpenApiConfiguration.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+@Configuration
+class OpenApiConfiguration(
+  buildProperties: BuildProperties,
+  val serviceUrlHelper: ServiceUrlHelper,
+) {
+  private val buildName: String = buildProperties.name
+  private val buildVersion: String = buildProperties.version
+
+  @Value("\${info.app.description}")
+  private val description: String = "VSIP Orchestration service used by VSIP frontend to make external API calls and collate responses."
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/config/OpenApiConfiguration.kt:33]: description update needed.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/config/ServiceUrlHelper.kt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.notificationsalertsvsip.config
+
+import org.springframework.stereotype.Component
+
+@Component
+class ServiceUrlHelper {
+  val firstPartUrl = "https://hmpps-notifi-alerts-vsip"
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/config/ServiceUrlHelper.kt:7]: is it notifi or notifications?

Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/config/ServiceUrlHelper.kt:7]: Thats the url name, for the server

Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/config/ServiceUrlHelper.kt:7]: It's this https://hmpps-notifi-alerts-vsip-dev.prison.service.justice.gov.uk

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Pull request merged**